### DISCUSSION TITLE
账户即将过期，微信支付成功后，账户仍显示即将过期

### DISCUSSION THREAD

**Discussion by lwjlhn** [getlantern/lantern-forum-cn#687] [Q&A]
请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。

**Comment by MaxMKMax** [comment]
@lwjlhn 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Comment by lwjlhn** [comment]
我昨天已经在这个链接里提交了

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年07月06日 00:27 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | lwjlhn ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 账户即将过期，微信支付成功后，账户仍显示即将过期 (Discussion #687) |

@lwjlhn 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!
